Games/League of Legends/BETA Client/script.js

```diff
@@ -1,32 +1,61 @@
 include(["Functions", "QuickScript", "CustomInstallerScript"]);
+include(["Functions", "Shortcuts","Wine"]);
 include(["Functions", "Verbs", "sandbox"]);
+include(["Functions", "Verbs", "d3dx9"]);
 include(["Functions", "Verbs", "vcrun2015"]);
 
+// Installs League of Legends and creates two shortcuts.
+//
+// One shortcut called League of Legends which starts the launcher
+// and one shortcut called BETA Client which starts the BETA client
+//
+// Important: 
+// The BETA-Client has to be installed using the launcher.
+// After the upgrade the BETA Client will do the game patching.
+
 new CustomInstallerScript()
     .name("League of Legends BETA Client")
     .editor("Riot Games")
     .applicationHomepage("http://euw.leagueoflegends.com/")
-    .author("Plata")
+    .author("Plata, feanor12")
     .installationCommand(function(wizard) {
-        return {command: "msiexec", args: ["/i", "C://LoL/Lol.EUW.msi","APPDIR=C:\\LoL","/q"]};
+        return {command: "msiexec", args: ["/i", "C://LoL_tmp/Lol.EUW.msi","APPDIR=C:\\LoL","/q"]};
     })
     .category("Games")
     .wineDistribution("staging")
     .wineVersion("2.0")
     .preInstall(function(wine, wizard) {
-        wine.sandbox();
-
+        //Download the setup file
         var setupFile = new Resource()
             .wizard(wizard)
             .url("https://riotgamespatcher-a.akamaihd.net/ShellInstaller/EUW/LeagueofLegends_EUW_Installer_2016_11_10.exe")
             .checksum("cbbf9883a59d2dada87e6aa8cc4484b07ea797da")
             .name("LeagueofLegends_EUW_Installer_2016_11_10.exe")
             .get();
-
+        //Create the temporary directory
```

Comment:
In general, this sort of comment is not required: a good code should speak by itself. 

A small tip that can help you: 
Ask yourself if you are describing a general behavior or if you are explaining your code. 
* In the first case, as pointed out by @plata, you should comment your code.
* In the second case, either the comment is redundant, either your code is not clear enough to be directly understandable. If you want to make your code better, a very good method is to create functions with a name explaining what t does instead of writing comments